### PR TITLE
His Grace is a better item and less globally-announced

### PR DESCRIPTION
**Pull request by ChangelingRain**
:cl: Joan
rscdel: His Grace no longer globally announces when He is awakened or falls to sleep.
rscdel: His Grace is not a toolbox, even if He looks like one.
experiment: His Grace no longer requires organs to awaken.
tweak: His Grace now gains 4 force for each victim consumed, always provides stun immunity, and will, generally, take longer to consume His owner.
experiment: His Grace must be destroyed to free the bodies within Him.
experiment: Dropping His Grace while He is awake will cause you to suffer His Wrath until you hold Him again.
rscadd: His Grace becomes highly aggressive after consuming His owner, and will hunt His own prey.
/:cl:

Notably, even if you allow His Grace to sleep, He will still gain bloodlust and have force for each person consumed. Dropping Him or throwing Him into a group of people may sometimes be a wise decision.

### CODE REVIEW

File: code/datums/status_effects/buffs.dm [hunk 1]
```diff
@@ -151,3 +151,49 @@
...
+	for(var/obj/item/weapon/his_grace/HG in owner.held_items)
+		if(HG.bloodthirst > bloodlust)
+			bloodlust = HG.bloodthirst
+	if(!bloodlust)
+		qdel(src)
+		return
+	var/grace_heal = Floor(bloodlust * 0.08)
+	owner.adjustBruteLoss(-grace_heal)
+	owner.adjustFireLoss(-grace_heal)
+	owner.adjustToxLoss(-grace_heal)
```
Comment (coiax) [inline code/datums/status_effects/buffs.dm]: Owner could be TOXINLOVING. We really need some sort of standard proc or flag for ignoring this sort of stuff. Tbh, I've never been a fan of TOXINLOVING, it makes no sense.

File: code/game/objects/items/weapons/his_grace.dm [hunk 2]
```diff
@@ -0,0 +1,190 @@
+//His Grace is a very special weapon granted only to traitor chaplains.
+//When awakened, He thirsts for blood and begins ticking a "bloodthirst" counter.
+//The wielder of His Grace is immune to stuns and gradually heals.
+//If the wielder fails to feed His Grace in time, He will devour them and become incredibly aggressive.
+//Leaving His Grace alone for some time will reset His thirst and put Him to sleep.
+//Using His Grace effectively requires extreme speed and care.
+/obj/item/weapon/his_grace
```
Comment (coiax) [inline code/game/objects/items/weapons/his_grace.dm]: /weapon/ is an empty datum class, plz remove

**Comment by coiax** [comment]
Another @Xhuis feature eaten by @ChangelingRain.

**Comment by Xhuis** [comment]
NO. YOU AREN'T TAKING THIS FROM ME. I ALREADY HAVE A PR HALFWAY THROUGH. I WILL MAKE IT.

File: code/datums/status_effects/buffs.dm [hunk 3]
```diff
@@ -151,3 +151,49 @@
 		return
 	playsound(cyborg, 'sound/effects/light_flicker.ogg', 50, 1)
 	cyborg.cell.give(power_to_give)
+
+/datum/status_effect/his_grace
+	id = "his_grace"
```
Comment (Fox-McCloud) [inline code/datums/status_effects/buffs.dm]: Side note, what's up with the awfully displayed tabbing on TG these days?

Comment (lzimann) [inline code/datums/status_effects/buffs.dm]: uh what?

**Comment by Robustin** [comment]
This could make it insanely OP but id rather see it be usable and then nerfed down than basically an elaborate suicide item.

**Comment by Armhulen** [comment]
to be fair it is a fun suicide item

**Comment by Xhuis** [comment]
The original premise of His Grace was kind of to be insanely OP, honestly (on Goon, that is). That's the best part.

**Comment by Steelpoint** [comment]
I'm pretty sure Goonstation removed 'His Grace' because it become way too powerful, even by Goon standards.

I don't personally mind if His Grace is a powerful item, considering its a 20 TC item, but it should not be powerful enough to wipe out the station.

**Comment by ChangelingRain** [comment]
Well, yes, His Grace should be extremely powerful. All you need to do to kill His owner is run away and wait a bit.

**Comment by Xhuis** [comment]
@Steelpoint Goon removed His Grace because it was even sillier than what we have now. Whenever someone examined it, they became a _slave_ to it and got an objective to feed it, so people would kill its owner, examine it, and go on a rampage; thus the cycle continued.

**Comment by Steelpoint** [comment]
That was also a contributing factor to its removal. 

Overall I'm fine with this item buff, even though its really more you choosing a long, drawn out, suicide.

**Comment by Xhuis** [comment]
Long, drawn out, _hilarious_ suicide.

**Comment by Armhulen** [comment]
it's an abstract kind of suicide

File: code/datums/status_effects/buffs.dm (hunk 3 above)
Comment (Supermichael777) [inline code/datums/status_effects/buffs.dm]: por que?

**Comment by Alek2ander** [comment]
This looks like something that could do with logs and extended fingerprinting.

**Comment by optimumtact** [comment]
I don't know about the stun immunity

File: interface/stylesheet.dm [hunk 4]
```diff
@@ -135,6 +135,9 @@ h1.alert, h2.alert		{color: #000000;}
 .greentext				{color: #00FF00;	font-size: 3;}
 .redtext				{color: #FF0000;	font-size: 3;}
 .clown					{color: #FF69Bf;	font-size: 3; font-family: "Comic Sans MS", cursive, sans-serif; font-weight: bold;}
+.his_grace				{color: #15D512;	font-size: 3; font-family: "Courier New", cursive, sans-serif; font-style: italic;}
```
Comment (AnturK) [inline interface/stylesheet.dm]: Please use just use multiple classes instead of this copypaste

**Review by AnturK — CHANGES_REQUESTED** [review]
CSS mutilation needs to stop at some point.

**Comment by Militaires** [comment]
:-1: to making his grace a not-toolbox :(

**Review by AnturK — APPROVED** [review]

**Comment by Xhuis** [comment]
Joan steals my code and makes it better again. Oh, well. I'm excited to see this on the server. An AI still completely shuts it down, though, but if it proves a problem even with resisting it's influence we can look into it. Thanks, Joan, but I still hate you. >:C

**Pull request merged**